### DISCUSSION TITLE
Error when trying to save a file as a .u3d

### DISCUSSION THREAD

**Discussion by YumeSK** [cnr-isti-vclab/meshlab#1252] [Q&A]
Hello! 

I am trying to save a .obj file as a .u3d, to make it to a 3D PDF later.

But when I try that in MeshLab it says Unkown error (-2147483643)
![Bildschirmfoto 2022-05-03 um 11 23 37](https://user-images.githubusercontent.com/104828380/166430786-5dc05f86-2d61-4caa-8e69-6bc0ad98fbea.png)

I am using a Mac ver. 12.3.1 and MeshLab 2022.02
If you need any more info please tell me :D

Does anyone have an idea of how to to fix this error?

Thanks in Advance! ^^

**Comment by moritzmhmk** [comment]
Using the version "MeshLab2020.07-macos" the error does not appear for me on mac OS 12.6
